src/validateUrlParams.js

```diff
@@ -0,0 +1,27 @@
+const validCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const isValidUrlParams = (text, toCase) => {
+  const errors = [];
+
+  if (!text) {
+    errors.push({
+      message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
```

Comment:
You can create an object with all your default error messages and use them as reference